docs/cli.md

```diff
@@ -387,6 +387,7 @@ This command prints the activation script of an environment.
 pixi shell-hook
 pixi shell-hook --shell bash
 pixi shell-hook --shell zsh
+pixi shell-hook -s powershell
```

Comment:
wdyt about using long option here also?

Reply:
Yeah normally I would use the full version but the repetition made me change it.